src/components/AccomodationCard.css

```diff
@@ -0,0 +1,46 @@
+.acc-card{
+    border: none;
+}
+
+
+.acc-card-pic {
+    display: block;
+    text-align: center;
+    width: auto !important;
+    height: auto !important;
+    margin: -50px;
```

Comment:
Negative margins are considered bad practice, can you achieve the same styling without this? Ask if you need help with this